src/index.js

```diff
@@ -296,6 +296,21 @@ const ScanThng = {
     api.scopes.Operator.prototype.stopStream = stopStream;
     api.scopes.Operator.prototype.scan = scan;
   },
+  /**
+   * Put image on canvas, convert it and export as data URL.
+   *
+   * @param {*} imageData - Image data.
+   * @param {object} userOptions - User options, if any.
+   * @returns {Promise}
+   */
+  processImage: Media.processImage,
```

Comment:
Suggest `convertImage` as the exported name, and add docs for the return type (`{ image }`):

```
* @returns {Promise<object>} Promise resolving an object { image} containing the image as a data URL.
```